### PR TITLE
Add requested  descriptions in an OCA README.rst file

### PR DESCRIPTION
**Pull request by phschmidt**
OCA sprint - 06/2015:  Review of existing 'readme' files or add file if missing.

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Adding @phschmidt's Github handle to the OCA database. You should then be governed by Acsone's entity CLA.

File: account_banking_mandate/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+This module depends on :
+* account_payment
+
+This modules are parts of the OCA/bank-payment suite.
+
+Configuration
+=============
+
+Need more informations ....
```
Comment (StefanRijnhart) [inline account_banking_mandate/README.rst]: Left a TODO here?

File: account_banking_payment_transfer/README.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+* account_banking_payment_export
+
+This modules are parts of the OCA/bank-payment suite.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+ * create a transfert account who allow reconciliation : option "Allow Reconciliation" activated.
```
Comment (StefanRijnhart) [inline account_banking_payment_transfer/README.rst]: Typos
- transfer_t_ 
- that allows

File: account_banking_payment_transfer/README.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ * configure transfert account on payment mode. Go to the menu Accounting > Configuration > Miscellaneous > Payment Mode and complete the section "Transfer move settings".
```
Comment (StefanRijnhart) [inline account_banking_payment_transfer/README.rst]: transfer_t_

File: account_banking_mandate/README.rst (hunk 1 above)
Comment (StefanRijnhart) [inline account_banking_mandate/README.rst]: Or, a little more helpful: there is nothing to configure for this module.

**Comment by StefanRijnhart** [comment]
Thanks! Just some small nags here.

**Comment by phschmidt** [comment]
Thanks, I will correct soon the files.

File: account_banking_mandate/README.rst [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+This module depends on :
+* account_payment
+
+This modules are parts of the OCA/bank-payment suite.
+
+Configuration
+=============
+
+TODO
```
Comment (pedrobaeza) [inline account_banking_mandate/README.rst:27]: You can remove this part

File: account_banking_mandate/README.rst [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+You can setup mandates from the accounting menu or directly from a bank
+account.
+
+Installation
+============
+
+This module depends on :
+* account_payment
+
+This modules are parts of the OCA/bank-payment suite.
```
Comment (pedrobaeza) [inline account_banking_mandate/README.rst]: s/are parts/is part

File: account_banking_pain_base/README.rst [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+This module was started during the Akretion-Noviat code sprint of November
+21st 2013 in Epiais les Louvres (France).
+
+Installation
+============
+
+This module depends on :
+- account_banking_payment_export
+
+This modules are parts of the OCA/bank-payment suite.
```
Comment (pedrobaeza) [inline account_banking_pain_base/README.rst]: s/are parts/is part

File: account_banking_payment_export/README.rst [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+(this field becomes visible when account_direct_debit is installed).
+
+Installation
+============
+
+This module depends on :
+* account_payment
+* base_iban
+
+This modules are parts of the OCA/bank-payment suite.
```
Comment (pedrobaeza) [inline account_banking_payment_export/README.rst]: s/are parts/is part

File: account_banking_payment_transfer/README.rst [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+=====
+
+This module allows to reconcile transfer account and invoice by selecting on a payment order a payment mode with the option "transfer account" activated.
+
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline account_banking_payment_transfer/README.rst]: Remove this section

File: account_banking_sepa_credit_transfer/README.rst [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+PAIN version 001.001.03, so it's probably the version of PAIN that you should
+try first.
+
+Installation
+============
+
+This module depends on :
+* account_banking_pain_base
+
+This modules are parts of the OCA/bank-payment suite.
```
Comment (pedrobaeza) [inline account_banking_sepa_credit_transfer/README.rst]: s/are parts/is part

File: account_banking_sepa_credit_transfer/README.rst [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+=====
+
+Allow to export an payment order in SEPA xml file when the payment mode refers an SEPA export.
+
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline account_banking_sepa_credit_transfer/README.rst]: Remove this section

File: account_banking_sepa_direct_debit/README.rst [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+Installation
+============
+
+This module depends on :
+* account_direct_debit
+* account_banking_pain_base',
+* account_banking_mandate
+
+This modules are parts of the OCA/bank-payment suite.
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/README.rst]: s/are parts/is part

File: account_banking_sepa_direct_debit/README.rst [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+Usage
+=====
+
+To use this module, you must select this payment mode on a direct debit order (Menu :Accounting > Payment > Direct Debit orders)
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/README.rst]: Remove this section

File: account_banking_payment_transfer/README.rst [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+the bank statement reconciliation, the payment order moves to the done
+status).
+    
+Installation
+============
+
+This module depends on :
+* account_banking_payment_export
+
+This modules are parts of the OCA/bank-payment suite.
```
Comment (pedrobaeza) [inline account_banking_payment_transfer/README.rst]: s/are parts/is part

**Comment by pedrobaeza** [comment]
You also need to rebase

File: account_banking_payment_export/README.rst [hunk 14]
```diff
@@ -0,0 +1,91 @@
...
+Usage
+=====
+
+This module provides a menu to configure payment order types : Accounting > Configuration > Miscellaneous > Payment Export Types 
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
+
+Known issues / Roadmap
```
Comment (pedrobaeza) [inline account_banking_payment_export/README.rst]: Remove the empty section

**Comment by pedrobaeza** [comment]
Thanks for the changes.

:+1:

**Comment by adrienpeiffer** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:

**Pull request merged**